index.d.ts

```diff
@@ -21,7 +21,7 @@ declare module "express-validator" {
    */
   function ExpressValidator(options?: ExpressValidator.Options.ExpressValidatorOptions): express.RequestHandler;
 
-  export = ExpressValidator;
+  export default ExpressValidator;
```

Comment:
I don't think that's possible.

This lib [exports](https://github.com/ctavan/express-validator/blob/841944e40b5bbc48aa5a92287ccb8436b26e9e5c/lib/express_validator.js#L567) a default function that initializes it, but ES6 modules cannot directly export callable functions. This is the reason why TS forces you to import modules the CommonJS way (`import x = require('module')` ).